pkg/proxy/userspace/proxier.go

```diff
@@ -1117,13 +1133,6 @@ func iptablesFlush(ipt iptables.Interface) error {
 	return utilerrors.NewAggregate(el)
 }
 
-// Used below.
```

Comment:
What was the reason for moving this to the top of the file?

Reply:
they are "basically" constants, normally those would go at the top of a file. maybe would be better if i just consolidated these somewhere ?  its not like `127.0.0.1` needs a bespoke implementation for all 4 proxy types :) ?